apps/cowswap-frontend/src/modules/tradeWidgetAddons/containers/HighFeeWarning/index.tsx

```diff
@@ -35,27 +36,55 @@ export function HighFeeWarning({ readonlyMode }: HighFeeWarningProps) {
   if (!isHighFee) return null
 
   return (
-    <WarningContainer level={level} isDarkMode={darkMode}>
-      <div>
-        <AlertTriangle size={24} />
-        Costs exceed {level}% of the swap amount!{' '}
-        <HoverTooltip wrapInContainer content={<HighFeeWarningMessage feePercentage={feePercentage} />}>
-          <ErrorStyledInfoIcon />
-        </HoverTooltip>{' '}
-      </div>
+    <>
+      <InlineBanner
+        bannerType={level ? BannerTypeMap[level] : StatusColorVariant.Info}
+        orientation={BannerOrientation.Vertical}
+        noWrapContent
+        width="100%"
+        customContent={
+          account &&
+          !readonlyMode && (
+            <InlineWarningCheckboxContainer>
+              <input
+                id="fees-exceed-checkbox"
+                type="checkbox"
+                onChange={toggleFeeWarningAccepted}
+                checked={feeWarningAccepted}
+              />{' '}
+              Swap anyway
+            </InlineWarningCheckboxContainer>
+          )
+        }
+      >
+        Costs exceed {level}% of the swap amount
```

Comment:
This is what Anxo mentioned yesterday.
To be more transparent, we can display the actual percentage here.
The `feePercentage` is already returned by `useHighFeeWarning`, just need to use it here.